CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# 0.3.0 - 2015-08-07
+
+- **BREAKING:** Renamed `class` metadata property to `classes`. [[RFC 0003](https://github.com/refractproject/rfcs/blob/master/text/0003-class-rename.md)]
```

Comment:
Also, `parameters` got renamed to `hrefVariables`. And `attributes` got renamed to `data`.